source/nodejs/adaptivecards-visualizer/css/outlook.css

```diff
@@ -201,7 +201,7 @@ a.ac-anchor:visited:active {
 }
 
 .ac-input.ac-textInput, .ac-input.ac-numberInput, .ac-input.ac-dateInput, .ac-input.ac-timeInput, .ac-input.ac-multichoiceInput {
-    border: 1px solid #DDDDDD;
+    border: 1px solid #888888;
```

Comment:
is this the first shade of gray that complies with the contrast requirements? It looks like it's a big color change; @dclaux is there a reason why this color was chosen as the expected value?

https://www.color-hex.com/color/dddddd
https://www.color-hex.com/color/888888

Reply:
The color used to match Fluent UI. It doesn't anymore, Fluent now uses `rgb(96, 94, 92)`:

![image](https://user-images.githubusercontent.com/1334689/124783954-52f65e00-defa-11eb-91ff-61eb48d5aefb.png)